### ISSUE TITLE
SpaceDock updated

### ISSUE THREAD

**Issue by rsparkyc** [tatjam/KSP-EngineLight#6]
Did this repo change owners?  I see v1.5 on spacedock, but no update here

**Comment by tatjam** [comment]
Yep the source code here is outdated, this is the code before the ToXik-yogHurt update, he made a huge code refactor and added some very nice features. I suppose I should really update this repo.

Here is ToXik-tohHurt's source code: https://github.com/ToXik-yogHurt/EngineLightUnofficial

(The code here is from RealFuels compatibility update)
